src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -462,6 +462,16 @@ public String toString() {
 			this.retried = retried;
 		}
 
+		DisposableAcquire(DisposableAcquire parent) {
+			this.cancellations = parent.cancellations;
+			this.sink = parent.sink;
+			this.pool = parent.pool;
+			this.obs = parent.obs;
+			this.opsFactory = parent.opsFactory;
+			this.pendingAcquireTimeout = parent.pendingAcquireTimeout;
+			this.retried = !parent.retried;
```

Comment:
is it possible that the `DisposableAcquire` is created twice this way, meaning that the third instance would be back to `retried == false` ? ie here the state transition from "retried" to "not retried" is possible, but that doesn't sound relevant and all disposableAcquires derived from an original one should probably be considered as "retried", right?

Reply:
agree I will change it